lightbeam/lightbeam.py

```diff
@@ -166,7 +166,8 @@ def write_structured_output(self, command):
                 # failures.line_numbers are split each on their own line; here we remove those line breaks
                 content = re.sub(r'"line_numbers": \[(\d|,|\s|\n)*\]', self.replace_linebreaks, content)
                 fp.write(content)
-        self.logger.info(f"results written to {self.results_file}")
+    
+            self.logger.info(f"results written to {self.results_file}")
```

Comment:
(Added an indent here so it's only printed if a results-file was actually created.)